testing/plugins_integration/pytest_qt_integration.py

```diff
@@ -0,0 +1,5 @@
+def test_hello_world(qtbot):
```

Comment:
Wasn't this supposed to get removed?

Reply:
Yep, missed in https://github.com/pytest-dev/pytest/pull/7721/commits/780752b25f976c539f79175b29c43d2ccbc9591e.

Reply:
🤦

Reply:
Makes me think we should be checking for 100% coverage for the smoke tests